### PR TITLE
include NN in SCT1 benchmark

### PR DESCRIPTION
**Pull request by costachris**

### CODE REVIEW

File: experiments/SCT1_benchmark/NN_closure/config.jl [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+using StatsBase
+using LinearAlgebra
+using CalibrateEDMF
+using CalibrateEDMF.ModelTypes
+using CalibrateEDMF.DistributionUtils
+using CalibrateEDMF.ReferenceModels
+using CalibrateEDMF.ReferenceStats
+using CalibrateEDMF.LESUtils
+using CalibrateEDMF.TurbulenceConvectionUtils
+src_dir = dirname(pathof(CalibrateEDMF))
```
Comment (ilopezgp) [inline experiments/SCT1_benchmark/NN_closure/config.jl:13]: src_dir is no longer needed

**Review by ilopezgp — APPROVED** [review]

**Comment by charleskawczynski** [comment]
Should we be adding the PDFs into the repo? If we build up too much data in the repo then it'll affects users time to `git clone`

**Comment by ilopezgp** [comment]
These are pretty lighweight for now. I would say it is convenient to have them here for a while

**Pull request merged**